plugins/inputs/kafka_consumer/kafka_consumer_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/Shopify/sarama"
 	"github.com/stretchr/testify/require"
 	"github.com/testcontainers/testcontainers-go"
-	"github.com/testcontainers/testcontainers-go/wait"
+	kTest "github.com/testcontainers/testcontainers-go/modules/kafka"
```

Comment:
Not sure if we should use uppercase in import aliases... How about
```suggestion
	kafkacontainer "github.com/testcontainers/testcontainers-go/modules/kafka"
```